Engine/Source/Editor/UnrealEd/Classes/CookOnTheSide/CookOnTheFlyServer.h

```diff
@@ -367,8 +367,11 @@ class UNREALED_API UCookOnTheFlyServer : public UObject, public FTickableEditorO
 
 	/**
 	* Get any packages which are in memory, these were probably required to be loaded because of the current package we are cooking, so we should probably cook them also
-	*/
-	TArray<UPackage*> GetUnsolicitedPackages(const TArray<FName>& TargetPlatformNames) const;
+	*/	
+//AMCHANGE_begin
+//#AMCHANGE Set to virtual to change it in child class
+	virtual TArray<UPackage*> GetUnsolicitedPackages(const TArray<FName>& TargetPlatformNames) const;
```

Comment:
I guess there are some changes related to that in our code base?

Reply:
Yes, but I'm waiting until this is merged before I submit.